openpype/pipeline/workfile/path_resolving.py

```diff
@@ -265,6 +265,10 @@ def get_last_workfile_with_version(
         if not match:
             continue
 
+        if not match.groups():
+            output_filenames.append(filename)
+            continue
+
```

Comment:
A comment on this change.
When setting the template for Unreal as just `{project[name]}.{ext}`, I would get the following error:
```
WARNING:openpype.tools.launcher.window.LauncherWindow:Action launch failed.
Traceback (most recent call last):
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\tools\launcher\window.py", line 411, in run_action
    action().process(filtered_session)
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\tools\launcher\actions.py", line 115, in process
    **self.data
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\lib\applications.py", line 363, in launch
    return self.manager.launch(self.full_name, *args, **kwargs)
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\lib\applications.py", line 503, in launch
    return context.launch()
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\lib\applications.py", line 1167, in launch
    prelaunch_hook.execute()
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\hooks\pre_global_host_data.py", line 44, in execute
    prepare_context_environments(temp_data)
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\lib\applications.py", line 1656, in prepare_context_environments
    _prepare_last_workfile(data, workdir, modules_manager)
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\lib\applications.py", line 1759, in _prepare_last_workfile
    workdir, file_template, workdir_data, extensions, True
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\pipeline\workfile\path_resolving.py", line 311, in get_last_workfile
    workdir, file_template, fill_data, extensions
  File "C:\Users\pypeclub\Desktop\OP3.14.1\openpype\pipeline\workfile\path_resolving.py", line 268, in get_last_workfile_with_version
    file_version = int(match.group(1))
IndexError: no such group
```

This happened because the regex expression matched the project name, but could not get any group in that string.